dts/riscv/it8xxx2.dtsi

```diff
@@ -69,6 +69,221 @@
 				write-block-size = <4>;
 			};
 		};
+
+		pinctrl: pin-controller {
+			compatible = "ite,it8xxx2-pinctrl";
+			status = "okay";
+			label = "PINCTRL";
+		};
+
+		pinctrla: pinctrl@f01610 {
+			compatible = "ite,it8xxx2-pinctrl-func";
+			reg = <0x00f01610 8>;   /* GPCR */
+			label = "PINCTRLA";
+			gpios = <&gpioa 0 0>;
+			func3-gcr =     <NO_FUNC  NO_FUNC  NO_FUNC  NO_FUNC
+					 0xf02032 0xf02032 0xf016f0 0xf016f0>;
+			func3-en-mask = <0        0        0        0
+					 0x02     0x02     0x10     0x0C    >;
+			func4-gcr =     <NO_FUNC  NO_FUNC  NO_FUNC  NO_FUNC
+					 NO_FUNC  NO_FUNC  NO_FUNC  NO_FUNC >;
+			func4-en-mask = <0        0        0        0
+					 0        0        0        0       >;
+			#pinmux-cells = <2>;
+		};
+
+		pinctrlb: pinctrl@f01618 {
+			compatible = "ite,it8xxx2-pinctrl-func";
+			reg = <0x00f01618 8>;   /* GPCR */
+			label = "PINCTRLB";
+			gpios = <&gpiob 0 0>;
+			func3-gcr =     <0xf016f5 0xf016f5 NO_FUNC NO_FUNC
+					 NO_FUNC  NO_FUNC  NO_FUNC 0xf01600>;
```

Comment:
note: I suspect a similar structure as STM32/GD32 could be followed here. see https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/gigadevice/gd32f4xx/gd32f4xx.dtsi#L228-L248

Reply:
Done.